src/components/NamePicker.tsx

```diff
@@ -27,8 +27,8 @@ export default function NamePicker(): JSX.Element {
 
   const handleClick1 = (event: MouseEvent<HTMLElement>): void => {
     setAnchorEl(event.currentTarget);
-    if (anchorEl === null) {
-      setDisabled1(!disabled1);
+    if (!anchorEl!) {
```

Comment:
Why is the second `!` there? An `!` in that position is telling TS that you _know_ this is not undefined. But clearly you don't know, that's why you're checking in the `if`. :-)

Oh, it's not in the check below with `anchorEl2`, so this must be a typo.

Reply:
I'm embarrassed lol, it's a typo! Updated and thanks!